package/src/constructs/check-group.ts

```diff
@@ -74,7 +77,7 @@ export class CheckGroup extends Construct {
   }
 
   addChecks (pattern: string) {
-    const matched = glob.sync(pattern, { nodir: true })
+    const matched = glob.sync(pattern, { nodir: true, cwd: this.__checkFilePath })
```

Comment:
`__checkFIlePath` is a path rather than a directory. Maybe this should be `path.dirname(this.__checkFilePath)`.